units/crons.go

```diff
@@ -309,6 +309,31 @@ func PopulateIdleHostJobs(env evergreen.Environment) amboy.QueueOperation {
 	}
 }
 
+func PopulateLastContainerFinishTimeJobs(env evergreen.Environment) amboy.QueueOperation {
+	return func(queue amboy.Queue) error {
+		flags, err := evergreen.GetServiceFlags()
```

Comment:
I think this job shouldn't check the degraded mode flags. The flags are mostly used to disable certain parts of the system that it can function without for short periods of time, usually due to third-party outages. But we want to be continually running this